package.json

```diff
@@ -1,15 +1,25 @@
 {
   "name": "snyk-policy",
+  "version": "0.0.0",
   "description": "Snyk's policy parser and matching logic",
-  "main": "lib/index.js",
+  "files": [
+    "dist/lib",
+    "LICENSE",
+    "README.md"
+  ],
+  "main": "./dist/lib/index.js",
   "scripts": {
+    "build": "npm run clean && tsc",
+    "clean": "rm -rf ./dist",
+    "dev": "tsc -w",
+    "env": "node -e 'console.log(process.env, process.versions)'",
     "lint": "run-p --max-parallel=${JOBS:-6} --aggregate-output lint:*",
     "lint:eslint": "eslint -c .eslintrc lib",
     "lint:commits": "commitlint --from=HEAD~20",
     "format": "prettier --write '{lib,test}/**/*.?s'",
-    "check-tests": "! grep 'test.only' test/**/*.test.js -n",
-    "env": "node -e 'console.log(process.env, process.versions)'",
-    "tap": "tap test/**/*.test.js -R spec --timeout=60",
+    "prepack": "npm run build",
```

Comment:
`prepublish` is deprecated (and doesn't do what I thought it did). I've updated to use `prepack` instead https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts